rules_bazel/java/java_integration_test.bzl

```diff
@@ -0,0 +1,84 @@
+def _compare_with_goldens_test_impl(ctx):
```

Comment:
Just `integration_test` should be sufficient, since this is in the `java` directory.

Also, could we rename this method to something more "imperative-verb"-like, such as `diff_integration_goldens`?

Reply:
renamed the file to `integration_test.bzl` and the method name to `diff_integration_goldens_test` (bazel test requires the name to end with `_test`)